test/evergreen.yml

```diff
@@ -79,6 +79,49 @@ functions:
               export PATH=/opt/mongodbtoolchain/v4/bin:$PATH
             fi
 
```

Comment:
Can we add a comment explaining these are preparing for the bazel/EngFlow migration? The text from the PR description look perfect for this

Reply:
Done